templates/templated_email/account/password_reset.email

```diff
@@ -8,11 +8,11 @@
 {% blocktrans context "Password reset e-mail text" %}
 Hi!
 
-You're receiving this e-mail because you or someone else has requested a password for your user account at {{ domain }}.
+You're receiving this e-mail because you or someone else has requested a password for your user account at {{ absolute_url }}.
 It can be safely ignored if you did not request a password reset. Click the link below to reset your password.
 {% endblocktrans %}
 
-{{ protocol}}://{{ domain }}{% url 'account:reset-password-confirm' uidb64=uid token=token %}
+{{ absolute_url }}{% url 'account:reset-password-confirm' uidb64=uid token=token %}
```

Comment:
Could we generate the whole url on the backend?  Like, it's better now, but still joining two parts in the template.